GTG/gtk/editor/taskeditor.ui

```diff
@@ -4,403 +4,461 @@
   <object class="GtkWindow" id="TaskEditor">
     <property name="can_focus">False</property>
     <property name="title" translatable="yes">Task</property>
-    <property name="default_width">500</property>
-    <property name="default_height">400</property>
+    <property name="default_width">800</property>
+    <property name="default_height">500</property>
     <signal name="configure-event" handler="on_move" swapped="no"/>
-    <child>
-      <object class="GtkBox" id="vbox4">
+    <child type="titlebar">
+      <object class="GtkHeaderBar" id="headerbar">
         <property name="visible">True</property>
-        <property name="can_focus">False</property>
-        <property name="orientation">vertical</property>
+        <property name="show_close_button">True</property>
+        <property name="title" translatable="yes">Task</property>
         <child>
-          <object class="GtkToolbar" id="task_tb1">
+          <object class="GtkMenuButton" id="mark_as_done_editor">
```

Comment:
why not `mark_as_done_button` or `mark_as_done` ? No need to put there `_editor` suffix